### PR TITLE
feat(DramaCool): add watchasia.to domain

### PR DESCRIPTION
**Pull request by Zukuchi**
add 'watchasia.to' please. it's another domain of dramacool, i think it myt be the original because it's easier to find drama names there unlike the dramacool.vc one. Thank you
(also, i dont know how to code so I just put it like that but anyways, yeah)

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [ ] I linted the code by running `yarn format`
- [ ] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/D/DramaCool/metadata.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
 		"nl": "Dramacool - Bekijk Aziatische drama's, films en shows met Engelse ondertiteling op Dramacool in Full HD Gratis. Een coole site voor grote mensen."
 	},
 	"url": "dramacool.vc",
+	"url": "watchasia.to",
 	"regExp": "[0-9w]{0,6}.*dramacool.[a-z]+",
 	"version": "1.3.0",
```
Comment (theusaf) [inline websites/D/DramaCool/metadata.json:16]:
```suggestion
	"version": "1.3.1",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by theusaf** [comment]
Consider also adding the domain from #8723 too

File: websites/D/DramaCool/metadata.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
 		"nl": "Dramacool - Bekijk Aziatische drama's, films en shows met Engelse ondertiteling op Dramacool in Full HD Gratis. Een coole site voor grote mensen."
 	},
 	"url": "dramacool.vc",
+	"url": "watchasia.to",
```
Comment (darkvillager2) [inline websites/D/DramaCool/metadata.json:14]: huh

**Comment by theusaf** [comment]
Closing for inactivity. You can open a new PR after resolving existing conversatoins

**Pull request closed without merging**